lib/plugins/features/movies/controller.js

```diff
@@ -1,11 +1,15 @@
 'use strict';
 
-const Movie = require('../../../models/movie');
+const Movie    = require('../../../models/movie');
+const Location = require('../../../models/location');
+
+const LOCATION_MODEL_FETCH_OPTIONS = { require: true };
+const MOVIE_MODEL_FETCH_OPTIONS    = { withRelated: ['locations'] };
```

Comment:
In lob-api, we add a `RELATED` field to our models (see [postcards](https://github.com/lob/lob-api/blob/master/src/models/postcard.js#L254) for example), and then use that when fetching postcards ([e.g.](https://github.com/lob/lob-api/blob/master/src/plugins/features/postcards/controller.js#L76)). That way we have the list of related tables in one place when we need them, but we don't necessarily always fetch `withRelated`! Something to consider, but I think this works well fine for our purposes here!

Reply:
Ah! I love it!